provider/common/bootstrap.go

```diff
@@ -180,11 +181,41 @@ func BootstrapInstance(ctx environs.BootstrapContext, env environs.Environ, args
 		ImageMetadata:   imageMetadata,
 		StatusCallback:  instanceStatus,
 		CleanupCallback: statusCleanup,
-	})
-	if err != nil {
+	}
+	zones, err := startInstanceZones(env, startInstanceArgs)
+	if errors.IsNotImplemented(err) {
+		// No zone support, so just call StartInstance with
+		// a blank StartInstanceParams.AvailabilityZone.
+		zones = []string{""}
+	} else if err != nil {
 		return nil, "", nil, errors.Annotate(err, "cannot start bootstrap instance")
 	}
 
+	var result *environs.StartInstanceResult
+	for i, zone := range zones {
+		if zone != "" {
+			ctx.Infof("Attempting to create instance in availability zone %q", zone)
```

Comment:
I am not sure we need this at Info level. We reduced the amount of bootstrap logging. I think it would be sufficient to log with the hardware info a bit later on what zone was used.

Reply:
OK, will drop it. The providers log it at debug anyway.